tests/Microsoft.Identity.Test.LabInfrastructure/LabServiceApi.cs

```
@@ -45,7 +45,29 @@ public LabServiceApi(KeyVaultSecretsProvider keyVault)
             _keyVault = keyVault;
         }
 
-        private LabResponse GetLabResponseFromAPI(UserQueryParameters query)
+        private LabResponse GetLabResponseFromApi(UserQueryParameters query)
+        {
+            //Fetch user
+            string result = CreateLabQuery(query);
+
+            if (string.IsNullOrWhiteSpace(result))
```

CreateLabQuery should throw this exception.